Sources/Web5/Dids/Resolution/DIDResolver.swift

```diff
@@ -6,6 +6,7 @@ public enum DIDResolver {
 
     private static var methodResolvers: [String: DIDMethodResolver] = {
         let defaultResolvers: [DIDMethodResolver] = [
+            DIDDHT.Resolver(),
             DIDJWK.Resolver(),
             DIDIon.Resolver(),
```

Comment:
can prob kill this since no one will be running nodes

Reply:
Makes sense, removed here! [94abc9e](https://github.com/TBD54566975/web5-swift/pull/15/commits/94abc9e88755b2b10d2f5a3e64b8f4dee7c4e580)